jovian/utils/clone.py

```diff
@@ -95,8 +96,13 @@ def clone(slug, version=None, fresh=True, include_outputs=True, overwrite=False)
     log('Downloading files..')
     for f in gist['files']:
         if not f['artifact'] or include_outputs:
-            with open(f['filename'], 'wb') as fp:
-                fp.write(get(f['rawUrl']).content)
+            # Delete  kernalspec entry from .ipynb file 
+            if f['filename'].endswith('.ipynb'):
+                content = _sanitize_notebook(f['rawUrl'])
```

Comment:
you can change function definition and pass `get(f['rawUrl']).content` here

Reply:
Changed.